### DISCUSSION TITLE
How to report bugs

### DISCUSSION THREAD

**Discussion by papertek** [papertek/CyDJ#10] [Announcements]
hi, you should report bugs here by making a new discussion thread instead of escalating it into an issue.

to report a bug just send a screenshot of said bug and explain the process on getting it. explain what the bug is too :) Don't forget to mention what browser and device you're using!!!

**Comment by eljulidi1337** [comment]
okay so... the emote on the BR is always the same for me and its kinda cut off... I don't know how it got this way but if I refresh its still there. I refreshed once and it was gone. I refreshed again and it reappeared. don't know if I did this correctly thanks for improving the servers xqcL. https://cdn.discordapp.com/attachments/560461680979476480/925850958729195530/unknown.png

**Reply by papertek** [reply]
forgor to mention you need to make a new discussion. dont reply to this one. i'll make a new one for you
